subsys/pcd/Kconfig

```diff
@@ -7,26 +7,54 @@
 menu "Peripheral CPU DFU (PCD)"
 
 menuconfig PCD
-	bool "Peripheral CPU DFU Support"
-	default y if SOC_NRF5340_CPUNET && IS_SECURE_BOOTLOADER
+	bool "Peripheral CPU DFU Support (deprecated - use PCD_APP or PCD_NET)"
```

Comment:
Thanks for the contribution. Sorry for late response due to holidays.

I would prefer if we stop using `CONFIG_PCD` altogether so that the deprecation will be a matter of simply removing the kconfig option when its executed.

Also I removed the default values for the addresses since it is good to highlight that the value of these options need some investigation if you want to set them manually. Hence with my changes you would need to provide this through your `prj.conf`. Hopefully this will not be inconvenient. Note that you easily can pass kconfig options to a child image through the `child_image` folder in your application directory (see [multi image docs](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html?#image-specific-variables))

I added a [commit](https://github.com/hakonfam/fw-nrfconnect-nrf-1/commit/0c59cc5537d813acb81fe4f89dc14d51d3ce4078) that you can cherry pick and squash which contains my proposed changes.

Note that I can push this straight to your branch and drive this PR from now on if this is more convenient for you, let me know.

Reply:
All sounds good.  I'm happy for you to take over and drive this PR.  Thanks!